<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerUSnavModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerUSnavModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="fileMhaLabel">
       <property name="text">
        <string>File (.mha): </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ctkPathLineEdit" name="filePathLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Current Frame: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="currentFrameLabel">
       <property name="text">
        <string>0/0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_0">
       <property name="text">
        <string>Transform Status: </string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="transformStatusLabel">
       <property name="text">
        <string>Unknown</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Image Dimensions</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="imageDimensionsLabel">
       <property name="text">
        <string>0x0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Available Transforms: </string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="availableTransformsLabel">
       <property name="text">
        <string>No Transforms</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="MRImageLabel">
       <property name="text">
        <string>MR image: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="qMRMLNodeComboBox" name="MRImageNodeComboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLScalarVolumeNode</string>
        </stringlist>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="frameSlider">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="previousPushButton">
       <property name="text">
        <string>Previous Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextPushButton">
       <property name="text">
        <string>Next Frame</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="previousValidFrameButton">
       <property name="text">
        <string>Previous Valid Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextValidFrameButton">
       <property name="text">
        <string>Next Valid Frame</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="previousInvalidFrameButton">
       <property name="text">
        <string>Previous Invalid Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextInvalidFrameButton">
       <property name="text">
        <string>Next Invalid Frame</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="consoleTextEdit"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerUSnavModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRImageNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
